src/pages/api/sendEmail.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable prettier/prettier */
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const nodemailer = require('nodemailer');
```

Comment:
Mixing require and import statements can lead to inconsistencies. I would use consistent syntax. i.e `import nodemailer from 'nodemailer';`